libefitcp/tcp.c

```diff
@@ -387,9 +391,8 @@ static EFI_STATUS ip_configuration(UINT32 port, EFI_IPv4_ADDRESS *address)
 		}
 	}
 
-	memcpy(address, &ip_data.ConfigData.StationAddress, sizeof(*address));
-
-	return EFI_SUCCESS;
+	ret = memcpy_s(address, sizeof(*address), &ip_data.ConfigData.StationAddress, sizeof(*address));
```

Comment:
@JuhaVainio , take this line as example, after change, if "buf_len" is the same as "count", you should consider whether we should do the change.

Reply:
If and when we change from memcpy to memcpy_s we should do it everywhere and do not leave old code. Part of code cleanup forced by Klocwork.

Reply:
How about?
ret = memcpy_s(address, sizeof(*address), &ip_data.ConfigData.StationAddress, sizeof(ip_data.ConfigData.StationAddress));

Reply:
it is better now. but it is just an example. please check others.
I know this change is to make the static code scan happy. since we don't have much time, you can ignore my comments and go directly. but, please add a note in your code refine TODO-list and re-consider it later.